client/Packages/packages-lock.json

```diff
@@ -123,7 +120,7 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.nuget.newtonsoft-json": {
-      "version": "3.2.1",
+      "version": "3.0.2",
```

Comment:
Do we know for sure that 3.0.2 is a good version of Newtonsoft to use? What was the reason for downgrading from 3.2.1? If it just sort of happened automatically _AND_ it isn't required transitively by other legitimate dependencies, I would prefer to err on the side of keeping things the same.